src/components/enter-password/enter-password-controller.ts

```diff
@@ -216,9 +219,10 @@ export function enterPasswordPost(
             req.session.user.isLatestTermsAndConditionsAccepted,
           requiresTwoFactorAuth: userLogin.data.mfaRequired,
           isConsentRequired: req.session.user.isConsentRequired,
-          mfaMethodType: userLogin.data.mfaMethodType,
+          mfaMethodType: req.session.user.enterEmailMfaType,
```

Comment:
I don't necessarily have an opinion on the right thing here, but why is it that this is using the session, whereas requiresTwoFactorAuth is using the API response? `mfaMethodType` does come back in the API response, so it may be safer to use that and let the backend keep track of these things. As I say, not sure I have a strong opinion, so if it's a deliberate choice, cool

Reply:
Agreed, at this point in the journey we had the enterEmailMfa Type field populated, but there is no actual need to replace the field, as the functionality is the same. Would indeed be safer to leave the implementation as it was before.